src/viam/services/service_base.py

```diff
@@ -25,6 +25,14 @@ def __init__(self, name: str) -> None:
     def from_robot(cls, robot: "RobotClient", name: str) -> Self:
         """Get the service named ``name`` from the provided robot.
 
+        ::
+
+            # Assume that the connect function is written and will return a valid machine
+            robot = await connect()
```

Comment:
I think this would be more helpful for the general use case. It's how we do it in https://python.viam.dev/autoapi/viam/app/app_client/index.html
```suggestion
            async def connect() -> ViamClient:
                # Replace "<API-KEY>" (including brackets) with your API key and "<API-KEY-ID>" with your API key ID
                dial_options = DialOptions.with_api_key("<API-KEY>", "<API-KEY-ID>")
                return await ViamClient.create_from_dial_options(dial_options)

            robot = await connect()
```